sorachung/StoriesByUs
# Can View first chapter of stories

#### Changes Made
INSTALLED DOMpurify via npm

1. Added file `Chapter.js` to `client/src/components/chapters/` directory.
2. Added file `ChapterStoryInfo.js` to `client/src/components/chapters/` directory.
3. Added file `chapterManager.js` to `client/src/modules/` directory.
4. Added file `ChapterRepository.cs and IChapterRepository.cs` to `Repositories/` directory.
5. Added file `ChapterController` to `Controllers/` directory.

1. Modified file `ApplicationViews.js` to include `chapter routes` and functionality.
2. Modified file `StoryCard.js` to include `word count` and functionality.
3. Modified file `storyManager.js` to include `getting a specific story` and functionality.
4. Modified file `StoryRepository.cs and IStoryRepository.cs` to include `getting a specific story` and functionality.
5. Modified file `Story.cs` to include `word count` and functionality.
7. Modified file `01_StoriesByUs_CreateDb.sql and 01_StoriesByUs_CreateDb.sql` to include `word count column on chapter` and functionality.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout src-readStory
   ```
8. Open a new Terminal tab (⌘T) and navigate to the server directory.
9. Test app functionality.
   > log in
   > click browse in navbar
   > browse
   > click on title of one story
   > verify that the chapter shows
10. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.

closes #9
